src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -1,5 +1,6 @@
 package teammates.test.cases.datatransfer;
 
+import static junit.framework.TestCase.assertNotSame;
```

Comment:
You can define `assertNotSame()` in `BaseTestCase` and make use of `AssertJunit`. See how we have done for all `assert*` methods.

Reply:
Could you explain? Why do we do so?

Reply:
The main reason for doing this is that we don't want to call `AssertJunit.*` and import it in each test. In addition, this could let us changing the implementations of assertion methods easily.